test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4995,31 +5049,7 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
                                           'lib', 'python', 'altair',
                                           'pbs_hooks',
                                           'PBS_translate_mpp.HK')
-        self.jacs_hk = os.path.join(self.pbs_conf['PBS_HOME'],
```

Comment:
you need to restore this as it is used under shasta

Reply:
I am deleting only site hooks. So, i did not need to add these in delete_hooks() method.
I have changed function name to delete_site_hooks()